### DISCUSSION TITLE
/lib folder shall be added to the `unmanagedClasspath` in the `JavaModule`

### DISCUSSION THREAD

**Discussion by b-gyula** [com-lihaoyi/mill#1465] [Ideas]
In the `JavaModule` the jars in the `{module root}/lib` folder shall be added to the `unmanagedClasspath` like in sbt and some other build tools

**Comment by lefou** [comment]
Maybe, we could make it default for [SBT-compatible modules](https://com-lihaoyi.github.io/mill/mill/Common_Project_Layouts.html#_sbt_compatible_modules), when this is the default in sbt, but given how easy it is to enable this [documented feature](https://com-lihaoyi.github.io/mill/mill/Configuring_Mill.html#_unmanaged_jars) when needed, I'm not a fan of making it the default. 

What other build tools provide that feature by default?
